docs/sdk-docs/private-chain/README.md

```diff
@@ -0,0 +1,144 @@
+
+## Identifying the correct smart contract version
+Before starting your private network setup, you first need to locate the version of Light Client you want to use, and then locate the version of the smart contracts used in this version of the Light Client.
+
+You can easily find out the version of the smart contracts used in your version of the Light Client by checking the `raiden-contracts` submodule. To locate the submodule you need to navigate to the [GitHub] project page. Once you find yourself in the project page, locate the `raiden-ts` subdirectory and look for the `raiden-contracts` submodule *(something like`raiden-contracts @ dfc0da6`)*. Check if you can distinct the `raiden-contracts` version by the commit message. If not, you have to follow the submodule link to the contracts repository and locate the `setup.py` file. You can find the version of the Raiden Contracts, if you look for the version variable *(`VERSION = '0.33.3'`)*.
+
+After locating the contract version, you have to go to the [Raiden] and locate a version of the python client compatible with the specific version of the smart contracts. You can easily find the proper version by going to the [requirements.txt] on the `develop` branch. Provided that the python client uses a newer version of the Raiden Contracts, you can use the [`Blame`] interface to find a commit using the correct version. In the `Blame` interface pressing the `View blame prior to this change` icon button *(The icon displayed before the line number)* you can go through the commits that changed the smart contract version, until you locate a commit that matches the correct version;
+
+ 
+## Setting up the Development Environment
+With the commit or tag of a compatible python client, you can go through the [tutorial] to set up your private network.
+
+Assuming the `develop` branch uses a newer contract version, you might need to slightly deviate from the tutorial. More specific you need to check out the correct branch or tag after [cloning] Raiden.
+
+For example, if you need to you the `v0.200.0-rc2` tag, you need to first checkout the tag before continuing with the tutorial.
+
+```bash
+git checkout v0.200.0-rc2
+```
+
+At the end of the guide you should have `Geth` running a private chain, with the Raiden contracts deployed, and a Raiden node running.
+
+### Locating the Smart contracts
+Now you bootstrapped the network, and you have to locate the deployment files.
+
+The deployment files will point the SDK to the correct smart contracts.
+
+Look for `env/lib/python3.7/site-packages/raiden_contracts/data_${VERSION}/`, in the directory that contains your virtual environment *(the one you created while following the tutorial)*.
+
+You should find the two following files in this directory:
+
+ * `deployment_private_net.json`
+ * `deployment_services_private_net.json`
+
+Copy these files to your project's `src/deployment` and proceed with the SDK or dApp initialization.
+
+### Starting the SDK
```

Comment:
I believe this is missing instructions or a reference to instructions on where/how to clone and install the LC repo? Because if we don't to that first there is not 'deployment` folder.

Reply:
If you are just using the SDK you don't need to clone the repo though. You use the one from npm in your package json dependencies.

Reply:
Ok, but I do think we should add a link to or mention cloning the repo alternatively using npm. Because right now we provide nice step by step instructions and here we assume that the reader have downloaded or installed the SDK.

Reply:
Do we have a guide that shows how to use the SDK from npm?

Reply:
We have in the LC docs https://lightclient.raiden.network/docs/installing-sdk/

Reply:
Good, then we can add a link to that :)